README.md

```diff
@@ -32,6 +32,22 @@ If you use this package, please cite [our paper](http://dx.doi.org/10.1007/978-9
 }
 ```
 
+#### Quick Start with Docker
+To use `mbuild` in a jupyter-notebook that runs from a docker container with all the dependencies installed use the following command:
+
+```sh
+$ docker pull mosdef/mbuild:latest
+$ docker run -it --name mbuild -p 8888:8888 mosdef/mbuild:latest\
+  /opt/conda/envs/mbuild-docker/bin/jupyter notebook --no-browser --ip="0.0.0.0"
+```
+
+Alternatively, you can also start a Bourne shell directly:
+```sh
+$ docker run -it --name mbuild mosdef/mbuild:latest
```

Comment:
Yeah it’s just in the container. You can mount an external dir with the -v I think. I think it just needs to be explained a bit more.

Reply:
Its up to to user to decide whether or not he/she wants to mount a user volume. Since this is a quickstart guide, let's leave it like it is. We should add a docker tutorial separately in the documentation.

Reply:
Aside from anything super specific to this tutorial I would just point users to external Docker tutorials, the internet is full of them and some of them are pretty decent ([here](https://docker-curriculum.com/) and [here](https://www.youtube.com/watch?v=zJ6WbK9zFpI&feature=youtu.be), for example).